enterprise/models.py

```diff
@@ -711,15 +712,16 @@ def notify_enrolled_learners(self, catalog_api_user, course_id, users, admin_enr
         with mail.get_connection() as email_conn:
             for user in users:
                 login_or_register = 'register' if isinstance(user, PendingEnterpriseCustomerUser) else 'login'
-                destination_url = destination_url.format(login_or_register=login_or_register)
+                enterprise_url = enterprise_url.format(login_or_register=login_or_register)
                 utils.send_email_notification_message(
                     user=user,
                     enrolled_in={
                         'name': course_name,
-                        'url': destination_url,
+                        'url': enterprise_url,
```

Comment:
So will your change keep the functionality intact for cases that aren't bulk enrollment (if you search for usages of this method without the admin_enrollment=True)? The only other case is the self_enroll case ... I wonder if the `url` that was used before is still applicable to that case.

If we don't want to have to deal with too many changes or not affect those other cases, we can also not use the `url` param and instead use a new `enterprise_course_url` which is only used in the bulk enrollment template, which leaves the other use cases untouched.